src/nft-marketplace/components/navbar/Navbar.jsx

```diff
@@ -6,26 +6,24 @@ import { Link } from "react-router-dom";
 
 const Menu = () => (
   <>
-    <Link target="_blank" rel="noreferrer" to='https://7f4shq8oyfd.typeform.com/to/F86oVLhb'><p>Join Beta</p></Link>
-    <Link target="_blank" rel="noreferrer" to='https://usecapsule.com'><p>Website</p></Link>
-    <Link target="_blank" rel="noreferrer" to='https://docs.usecapsule.com/'><p>View Docs</p></Link>
-    <Link target="_blank" rel="noreferrer" to='https://github.com/capsule-org/capsule-example'><p>View Demo App Github</p></Link>
+    <Link target="_blank" rel="noreferrer" to='https://github.com/capsule-org/capsule-example'><p>Demo</p></Link>
```

Comment:
Any reason we are naming this "Demo"? It is routing to the Github code which some users may not have direct access to view

Reply:
It's in the design. Not sure where it should link, @nityas do you have a preference?

Reply:
Removed it for now